website/docs/brain/brain-install.md

```diff
@@ -3,37 +3,124 @@ id: brain-install
 title: Install
 ---
 
-# Farm-ng Brain ADK
+# Brain ADK Installation
+
+[![PyPI version](https://badge.fury.io/py/farm-ng-amiga.svg)](https://pypi.org/project/farm-ng-amiga)
 
 ## Install
 
-We recommend running the brain ADK applications in a virtual environment to avoid conflicts with other packages / versions installed on your system.
+:::info
+If you would like to test your installation by running the `farm-ng-amiga` [**examples**](https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples),
```

Comment:
maybe we should simplify and point only to the https version

Reply:
We can consider if current rec for ssh is causing more concerns